docs/docs/build/packages.mdx

```diff
@@ -3,29 +3,42 @@ title: Packages
 sidebar_position: 9
 ---
 
-The Fluid Framework is a multi-layered system consisting of dozens of individual npm packages. Most developers will only
-need two or three of these packages for typical Fluid development: The `fluid-framework` package, which contains the
-public API for the Fluid Framework, and a service-specific client package, such as the
-`@fluidframework/tinylicious-client` package.
+The Fluid Framework is a multi-layered system consisting of dozens of individual npm packages.
+Most developers will only need two or three of these packages for typical Fluid development:
+- The [fluid-framework][] package, which contains the public API for the Fluid Framework
+- A service-specific client package, such as the [@fluidframework/azure-client](../api/azure-client/index.md).
 
 ## Primary API: fluid-framework
 
-The `fluid-framework` package consists primarily of two portions: the [FluidContainer][] object and a selection of
+The [fluid-framework][] package consists primarily of two portions: the [FluidContainer][] object and a selection of
```

Comment:
Didn't know about this syntax. Cool!

Reply:
It has a downside though: If there is no corresponding definition in the document, Docusaurus will just render it as plain text :/